### ISSUE TITLE
Repository for Debian Bookworm

### ISSUE THREAD

**Issue by xtaran** [eLvErDe/hwraid#139]
It's that time of the year I guess. :-)

It's a little early for release, but it would be handy if there would be Bookworm repo for testing etc. Thank you for all your work!

(Text blatantly copied from #124 as it is fitting again every two years. And #124 can be closed now btw. ;-))

**Comment by PVasileff** [comment]
Bookworm is now stable. I have using this repo for megacli from it on my proxmox server that I want to uprade to proxmox 8 (based on Bookworm) but waiting for bookworm in http://hwraid.le-vert.net/debian

**Comment by 10RUPTiV** [comment]
@eLvErDe Any idea when you will have some time to make the bookworm repo ?

**Comment by Polve** [comment]
It seems like the repo is unmantained: some good samaritan should fork  it and update the community, thanking @eLvErDe for his good work in last years

**Comment by waja** [comment]
@polve thanks for the very usefull comment, but why not doing it yourself?

**Comment by Polve** [comment]
Unfortunately I don't have either the skills nor the time needed to do it!

**Comment by waja** [comment]
> Unfortunately I don't have either the skills not the time needed to do it!

So you comment was absolutely superfluous.

**Comment by Polve** [comment]
I don't think so: I informed the other posters waiting for a reply for the current mantainer.
Why such animosity, is there any problem?

**Comment by aque** [comment]
Does the Bullseye .deb work under Bookworm?

**Comment by plegrand1** [comment]
I use them on four freshly upgraded debian (Bookworm) and it seems to work

**Comment by tsamu** [comment]
> I use them on four freshly upgraded debian (Bookworm) and it seems to work

For me it`s ok too for a newly installed Bookworm.

If you follow setup instructions (http://hwraid.le-vert.net/wiki/DebianPackages), you get keyring warning:

`W: http://hwraid.le-vert.net/debian/dists/bullseye/Release.gpg: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.`

To solve this issue, remove keyring and add it properly.

`apt-key list`

Find HWRaid

`apt-key del "0073 .... D3B4"`


`curl https://hwraid.le-vert.net/debian/hwraid.le-vert.net.gpg.key | gpg --dearmor | sudo tee /usr/share/keyrings/hwraid-archive-keyring.gpg >/dev/null`

/etc/apt/sources.list.d/hwraid.list
`deb [signed-by=/usr/share/keyrings/hwraid-archive-keyring.gpg] http://hwraid.le-vert.net/debian bullseye main
`

**Comment by bodik** [comment]
> curl https://hwraid.le-vert.net/debian/hwraid.le-vert.net.gpg.key | gpg --dearmor | sudo tee /usr/share/keyrings/hwraid-archive-keyring.gpg >/dev/null

i'd just suggest to place the key in `/etc/apt/trusted.gpg.d` instead of /usr/share

**Comment by Fantu** [comment]
Hi, someone know if there are similar and updated repository? from a fast search I didn't found

**Comment by plegrand1** [comment]
The new Bookworm repository no longer contains the "megacli" and "megaclisas-status" packages.
Only the "sas3flash" and "sas3ircu" packages are present.
Is this normal?
Is there an explanation for this?
Thanks for your help
Pascal

**Comment by mohag** [comment]
Ubuntu noble is the one that I'm missing...
